docs/INSTALLATION.md

```diff
@@ -0,0 +1,49 @@
+# Installation
+This will walk you through a simple 
```

Comment:
a simple what?

Reply:
resolved in db6d1ff9f835e6982b243ff053f6bf6321dd0e88